config/global.ini.php

```diff
@@ -1256,6 +1256,27 @@
 PluginsInstalled[] = Monolog
 PluginsInstalled[] = Intl
 
+[PluginPagePerformance]
```

Comment:
Our other plugins are using the plugin name as section only, so guess it makes sense to change that to `PagePerformance` only.